keyboards/scrabblepad/keymaps/random/keymap.c

```diff
@@ -0,0 +1,75 @@
+#include "scrabblepad.h"
+#include <stdlib.h>
+
+static uint16_t keystroke;
+static bool shift_pressed;
+
+static int lower = (int) KC_A;
+static int upper = (int) KC_Z;
+
+uint16_t random_keycode(void) {
```

Comment:
Since you're only generating basic keycodes, you should probably narrow the type to `uint8_t` here. Furthermore, `register_code` takes `uint8_t`.
```suggestion
uint8_t random_keycode(void) {
```